doc/source/releases/21.09_announce_user.rst

```diff
@@ -55,37 +55,80 @@ News Webhook
 
 You might be reading this in the recently added "News" feature! This is the very first iteration of this component, hopefully in the future we will expand it to bring you all of your news and Galaxy notifications. The idea with the current iteration is that whenever your server admin updates Galaxy to a new version, everyone using the server will get a small, non-intrusive notification with these release notes! There you can find out all of the new features in Galaxy and quick videos on their use. (`Pull Request 12396`_)
 
-
 New Datatypes
 ===========================================================
 
-.. datatypes
+* Added documentation for FASTQ datatypes and implemented quality check
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 11931`_
 * Make dataset preview for h5mlm datatype
   (thanks to `@qiagu <https://github.com/qiagu>`__).
   `Pull Request 11935`_
 * Add datatypes for Structural Materials Hexrd application
   (thanks to `@jj-umn <https://github.com/jj-umn>`__).
   `Pull Request 11957`_
+* Adding new subclass types
+  (thanks to `@maikenp <https://github.com/maikenp>`__).
+  `Pull Request 12097`_
+* Converters: use target datatype
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12185`_
 * Add bref3 datatype
   (thanks to `@gallardoalba <https://github.com/gallardoalba>`__).
   `Pull Request 12199`_
-* mothur datatype: PairwiseDistanceMatrix sniffer may be too unspecific
+* Converters: add missing tests and add linting to converter tests
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12202`_
+* Converters: Unify converters to tabix and bgzip
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12213`_
+* Converters: Unify molecules converters
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12214`_
+* Converters: Unify dcd, trr, xtc
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12224`_
+* converters: Unify bcf converters
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12225`_
+* Fix edta metadata setting
+  (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
+  `Pull Request 12273`_
+* Increase specificity of mothur.pair.dist sniffer
   (thanks to `@bernt-matthias <https://github.com/bernt-matthias>`__).
   `Pull Request 12280`_
 * Add "ExpressionSet RData object" Datatype
   (thanks to `@mtekman <https://github.com/mtekman>`__).
   `Pull Request 12336`_
+* Small fix in binary.py
```

Comment:
I'd previously manually removed these to focus only on new datatypes since these sort of changes seemed less important to highlight, so past release notes have been excluding these as well. If you feel it's important to have all these we could sort them maybe into datatypes and a subection on datatype updates to help readers skim the content most effectively?